package.json

```diff
@@ -61,7 +61,26 @@
           "jsxBracketSameLine": false
         }
       ]
-    }
+    },
+    "overrides": [{
+      "files": "app/config/config-default.js",
+      "rules": {
+        "prettier/prettier": [
+          "error",
+          {
+            "printWidth": 120,
+            "tabWidth": 2,
+            "singleQuote": true,
+            "trailingComma": "es5",
+            "bracketSpacing": false,
+            "semi": true,
+            "useTabs": false,
+            "parser": "babylon",
+            "jsxBracketSameLine": false
+          }
```

Comment:
Didn't find a way to avoid duplicating prettier options here, tried with `.prettierrc` file and ESLint comments. Prettier seems to always merge with the default options.